pkg/local_workflows/report_analytics_workflow.go

```diff
@@ -81,6 +81,8 @@ func reportAnalyticsEntrypoint(invocationCtx workflow.InvocationContext, inputDa
 			workflow.NewTypeIdentifier(WORKFLOWID_REPORT_ANALYTICS, reportAnalyticsWorkflowName),
 			"application/json",
 			[]byte(commandLineInput),
+			workflow.WithConfiguration(&config),
```

Comment:
Suggestion: Let's limit impact of the changes and not use this function everywhere.